wpilib/wpilib/compressor.py

```diff
@@ -148,14 +152,34 @@ def getSmartDashboardType(self):
         return "Compressor"
 
     def initTable(self, subtable):
-        self.table = subtable
-        self.updateTable()
-
-    def getTable(self):
-        return self.table
+        if subtable is not None:
+            self.enabledEntry = subtable.getEntry("Enabled")
+            self.pressureSwitchEntry = subtable.getEntry("Pressure Switch")
+        else:
+            self.enabledEntry = None
+            self.pressureSwitchEntry = None
 
     def updateTable(self):
-        table = self.getTable()
-        if table is not None:
-            table.putBoolean("Enabled", self.enabled())
-            table.putBoolean("Pressure Switch", self.getPressureSwitchValue())
+        if self.enabledEntry is not None:
+            self.enabledEntry.setBoolean(self.enabled())
+        if self.pressureSwitchEntry is not None:
+            self.pressureSwitchEntry.setBoolean(self.getPressureSwitchValue())
+
+    def startLiveWindowMode(self):
```

Comment:
Hm? Isn't this done in liveWindowSendable?

Reply:
what? setting up the listener? wrong entry

Reply:
Well, only one entry is on the LiveWindow here. Simple fix.

Reply:
?